cc_licenses/settings/base.py

```diff
@@ -206,3 +224,6 @@
 
 # django-distill settings
 DISTILL_DIR = f'{ROOT_DIR}/build/'
+
+# By default, assume the translation data's repo is checked out beside this one.
+TRANSLATION_REPOSITORY_DIRECTORY = os.path.join(ROOT_DIR, "..", "cc-licenses-data")
```

Comment:
I think we should change `TRANSLATION_REPOSITORY_DIRECTORY` to an environment variable. If no path is supplied for the translation repo then default to `os.path.join(ROOT_DIR, "..", "cc-licenses-data")`. I think this will be more consistent with what we are doing in `cc-link-checker`.

Reply:
Will do.